Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/Element.cs

```diff
@@ -85,18 +93,21 @@ public void Click()
         public string GetAttribute(string attr)
         {
             LogElementAction("loc.el.getattr", attr);
-            return DoWithRetry(() => GetElement().GetAttribute(attr));
+            var value = DoWithRetry(() => GetElement().GetAttribute(attr));
+            LogElementAction("loc.el.attr.value", attr, value);
+
+            return value;
         }
 
         public virtual RemoteWebElement GetElement(TimeSpan? timeout = null)
         {
             try
             {
-                return CacheConfiguration.IsEnabled 
+                return CacheConfiguration.IsEnabled
                     ? Cache.GetElement(timeout)
                     : (RemoteWebElement) Finder.FindElement(Locator, elementState, timeout);
             }
-            catch (NoSuchElementException ex)
+            catch (NoSuchElementException ex) when (LoggerConfiguration.LogPageSource)
```

Comment:
Wiil it throw exception If LoggerConfiguration.LogPageSource is false?

Reply:
Of course, why not? the exception would not be caught, but it is thrown only by Finder

Reply:
`when` keyword says that exception will be handled only when the expression is true

Reply:
yes, and our "handling" is just logging and rethrowing. So, if LoggerConfiguration.LogPageSource is false, then we just not handle it.

Reply:
@pavelanihimovsky I've added 2 tests for LogPageSource functionality, to prove that it works as expected